src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1064,7 +1079,7 @@ private IParameterInitializer CreateBoundParameterEqualsValueOperation(BoundPara
         private IBlockStatement CreateBoundBlockOperation(BoundBlock boundBlock)
         {
             Lazy<ImmutableArray<IOperation>> statements =
-                new Lazy<ImmutableArray<IOperation>>(() => boundBlock.Statements.Select(s => Create(s)).Where(s => s.Kind != OperationKind.None).ToImmutableArray());
+                new Lazy<ImmutableArray<IOperation>>(() => boundBlock.Statements.Select(s => Create(s)).Where(s => s != null && s.Kind != OperationKind.None).ToImmutableArray());
```

Comment:
Where are we returning `null` for a BoundNode? This seems like a bug.

Reply:
The CSharpOperationFactory.Create(BoundNode boundNode) method returns null if the boundNode parameter is null. This could happen when there is an optional operation child, like Instance property of IMemberReferenceExpression.